frontend/src/modules/scaffold/backboneFormsOverrides.js

```diff
@@ -1,292 +1,325 @@
-
-		return {
-			help: schema.help || '',
-			title: schema.title,
-			titleHTML: schema.titleHTML,
-			fieldAttrs: schema.fieldAttrs,
-			editorAttrs: schema.editorAttrs,
-			key: this.key,
-			editorId: this.editor.id,
-			legend: schema.legend,
-			fieldType: this.schema.fieldType
-		};
-	};
-
-	Backbone.Form.editors.List.Modal.prototype.itemToString = function(value) {
-		var createTitle = function(key) {
-			var context = { key: key };
-
-			return Backbone.Form.Field.prototype.createTitle.call(context);
-		};
-
-		value = value || {};
-
-		//Pretty print the object keys and values
-		var parts = [];
-		_.each(this.nestedSchema, function(schema, key) {
-			
-			var desc = schema.title ? schema.title : createTitle(key),
-			val = value[key];
-
-			// If value is an array then print out how many items it contains
-			if (_.isArray(val)) {
-				var itemsString = ' items';
-				if (val.length === 1) {
-					itemsString = ' item';
-				}
-				val = val.length + itemsString;
-			} else if (_.isObject(val)) {
-				// If the value is an object print out the keys and values
-				valueText = '';
-				_.each(val, function(value, key) {
-					valueText += '<br>' + key + ' - ' + value;
-				});
-				val = valueText;
-			}
-
-			if (_.isUndefined(val) || _.isNull(val)) val = '';
-
-			parts.push('<b>' + desc + '</b>: ' + val);
-
-		});
-
-		return parts.join('<br />');
+define([
+  'core/origin',
+  'backbone-forms',
+  'backbone-forms-lists',
+], function(Origin, BackboneForms, BackboneFormsLists) {
+
+  var templates = Handlebars.templates;
+  var fieldTemplate = templates.field;
+  var templateData = Backbone.Form.Field.prototype.templateData;
+  var listSetValue = Backbone.Form.editors.List.prototype.setValue;
+  var textInitialize = Backbone.Form.editors.Text.prototype.initialize;
+  var textAreaRender = Backbone.Form.editors.TextArea.prototype.render;
+  var textAreaSetValue = Backbone.Form.editors.TextArea.prototype.setValue;
+
+  Backbone.Form.prototype.constructor.template = templates.form;
+  Backbone.Form.Fieldset.prototype.template = templates.fieldset;
+  Backbone.Form.Field.prototype.template = fieldTemplate;
+  Backbone.Form.NestedField.prototype.template = fieldTemplate;
+  Backbone.Form.editors.List.prototype.constructor.template = templates.list;
+  Backbone.Form.editors.List.Item.prototype.constructor.template = templates.listItem;
+
+  // add reset to default handler
+  Backbone.Form.Field.prototype.events = {
+    'click [data-action="default"]': function() {
+      this.setValue(this.schema.default);
+
+      return false;
+    }
+  };
+
+  // merge schema into data
+  Backbone.Form.Field.prototype.templateData = function() {
+    return _.extend(templateData.call(this), this.schema);
+  };
+
+  // process nested items
+  Backbone.Form.editors.List.Modal.prototype.itemToString = function(value) {
+    var createTitle = function(key) {
+      var context = { key: key };
+
+      return Form.Field.prototype.createTitle.call(context);
```

Comment:
I think this should be: `Backbone.Form.Field.prototype.createTitle.call(context);`